docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -97,7 +100,16 @@ lockScreenConfig.setAppIcon(appIconInt); // For example, R.drawable.lockscreen i
 !@
 
 ### Showing the OEM Logo
-The default lock screen handles retrieving and setting the OEM logo from head units that support this feature. This feature can be disabled on the default lock screen by setting `showDeviceLogo` to false. 
+The default lock screen handles retrieving and setting the OEM logo from head units that support this feature.
+
+@![iOS]
+![Custom Lock Screen](assets/OEMLogoLockScreen.png)
+!@
+@![android]
+![Custom Lock Screen](assets/OEMLogoLockScreen_Android.png)
```

Comment:
Not sure where my previous comment went...

Can you update the app's logo in "OEMLogoLockScreen.png" and "OEMLogoLockScreen_Android.png" to use the same app icon you used in the "CustomLockScreen.png" screenshots? I think it is confusing to have two SDL icons in the screenshot.

Also, can you remove the "Swipe down to dismiss..." message from the Android screenshot.

Reply:
Updated screenshots in most recent commit.